### ISSUE TITLE
Tekstuele issues op pagina 95

### ISSUE THREAD

**Issue by IwanKlein** [BROprogramma/geotechnischsondeeronderzoek#43]
" De hoogte is onbekend, bij transformatie is uitgegaan van 0 m NAP."
Opmerking: Wordt er op enige manier aangegeven dat hiervan is uitgegaan bij een betreffende sondering? Nu is dit namelijk inzichtelijk door de vermelding t.o.v. MV.

**Comment by MarcelReuvers** [comment]
Ja, de transformatiemethode wordt altijd vastgelegd en uitgegeven. Verder is de omschrijving van de domeinwaarde RDNAPTRANSMV0  aangepast : ..de verticale positie van het maaiveld is onbekend …
